src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.cs

```diff
@@ -22,25 +22,6 @@ public partial class PlatformCompatabilityAnalyzerTests
     {
         private const string s_msBuildPlatforms = "build_property._SupportedPlatformList=windows,browser,macOS,maccatalyst, ios, linux;\nbuild_property.TargetFramework=net5.0";
 
-        [Fact(Skip = "TODO need to be fixed: Test for for wrong arguments, not sure how to report the Compiler error diagnostic")]
-        public async Task TestOsPlatformAttributesWithNonStringArgumentAsync()
```

Comment:
This is unrelated, though removing this test as we have an analyzer that explicitly checking the wrong  arguments
https://github.com/dotnet/roslyn-analyzers/blob/main/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/UseValidPlatformString.cs

Reply:
Removed all unrelated code